aten/src/ATen/templates/TensorBody.h

```diff
@@ -352,52 +352,52 @@ class TORCH_API Tensor {
   }
 
   /// Returns a `Tensor`'s layout. Defined in Type.h
-  Layout layout() const noexcept;
+  Layout layout() const noexcept { return impl_->layout(); }
```

Comment:
nit: does this match the style of the rest of the header? Usually I've seen implementations on a separate line.